app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/EditDietFragment.java

```diff
@@ -0,0 +1,163 @@
+package openfoodfacts.github.scrachx.openfood.fragments;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentTransaction;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Switch;
+
+import java.util.Locale;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import openfoodfacts.github.scrachx.openfood.R;
+import openfoodfacts.github.scrachx.openfood.models.Diet;
+import openfoodfacts.github.scrachx.openfood.models.DietName;
+import openfoodfacts.github.scrachx.openfood.repositories.DietRepository;
+import openfoodfacts.github.scrachx.openfood.repositories.IDietRepository;
+
+public class EditDietFragment extends Fragment {
+    // TODO: Rename parameter arguments, choose names that match
+    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+    private static final String ARG_DIET_NAME = "dietName";
+    //Récupération des éléments du fragment de saisie d'une Diet
+    @BindView(R.id.diet_name)
+    EditText dietName;
+    @BindView(R.id.diet_description)
+    EditText dietDescription;
+    @BindView(R.id.diet_enabled)
+    Switch dietEnabled;
+    @BindView(R.id.ingredients_authorised)
+    EditText ingredientsAuthorised;
+    @BindView(R.id.ingredients_soso)
+    EditText ingredientsSoSo;
+    @BindView(R.id.ingredients_unauthorised)
+    EditText ingredientsUnauthorised;
+    @BindView(R.id.save_edits)
+    Button saveEdits;
+    //Récupération de la langue (théorique) de saisie :
```

Comment:
```suggestion
    // Fetching of the (theoretical) language of input:
```

Reply:
Do you want me to translate my "Log" and comments ?